crates/services/relayer/src/storage.rs

```diff
@@ -9,16 +13,19 @@ use fuel_core_storage::{
     },
     kv_store::StorageColumn,
     structured_storage::TableWithBlueprint,
-    transactional::Transactional,
+    transactional::{
+        Modifiable,
+        StorageTransaction,
+    },
     Error as StorageError,
     Mappable,
     Result as StorageResult,
     StorageAsMut,
-    StorageAsRef,
     StorageMutate,
 };
 use fuel_core_types::{
     blockchain::primitives::DaBlockHeight,
+    fuel_merkle::storage::StorageInspect,
```

Comment:
This looks like a bad auto-import - you probably want `StorageInspect` from `fuel_core_storage`